ruby-challenges.rb

```diff
@@ -19,17 +28,58 @@
 nums_array2 = [25, 17, 47, 11]
 # Expected output: 100
 
+def sum_of (nums_array)
+    nums_array.inject(:+)
+end
 
+p sum_of nums_array1
+p sum_of nums_array2
+# Output:
+# 76
+# 100
 
 # --------------------3a) Create a class called Bike that is initialized with a model, wheels, and current_speed. The default number of wheels is 2. The current_speed should start at 0. Create a bike_info method that returns a sentence with all the data from the bike object.
 
 # Expected output example: 'The Trek bike has 2 wheels and is going 0 mph.'
 
-
+class Bike
+    attr_accessor :model, :wheels, :current_speed
+      def initialize(model, wheel, current_speed)
```

Comment:
Stellar developer, in case you want to challenge yourself. Try hard-coding the current speed on line 47. And then  on line 67 and 68 just pass the argument of the new model name.
Another technique you can use, on line 47 just use one argument of the model. Then hard code your wheels and current speed.